src/transformers/modeling_utils.py

```diff
@@ -1231,40 +1287,31 @@ def _generate_no_beam_search(
             outputs = self(**model_inputs)
             next_token_logits = outputs[0][:, -1, :]
 
+            next_token_logscores = self.finalize_generation_logscores(
+                next_token_logits,
+                input_ids,
+                batch_size,
+                1,
```

Comment:
comment num_beams (cause it looks wierd :))

Reply:
I think we should work with keyword arguments `next_token_logits=next_token_logits`, ... => it's less error prone.